queries.md

```diff
@@ -5,28 +5,65 @@
 ## 1. Find all the companies that include 'Facebook' on the **name** field.
 
  - **`query`**: {name: 'Facebook'}
+ - **`query`**: 
+- **`projection`**: 
  
  ## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
 
  - **`query`**: {category_code: 'web'}
  - **`projection`**: {name: 1, _id: 0}
 
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
+- **`query`**: {name: "Twitter"}
+- **`projection`**: {name: 1, category_code:1, founded_year:1}
```

Comment:
Here, don't forget to specify "_id: 0", or it's going to be retrieved by default! :)